extensions/wikia/AdEngine/js/lookup/rubiconFastlane.js

```diff
@@ -195,6 +195,7 @@ define('ext.wikia.adEngine.lookup.rubiconFastlane', [
 		rubicon.src = rubiconLibraryUrl;
 
 		node.parentNode.insertBefore(rubicon, node);
+		context = adContext.getContext();
```

Comment:
Because [this context](https://github.com/Wikia/app/blob/d9b83cb6a657deb0da77ea6b34ff02c88243b0ea/extensions/wikia/AdEngine/js/lookup/rubiconFastlane.js#L55) is too early for mercury